src/43F0.c

```diff
@@ -1,7 +1,7 @@
 #include "common.h"
 #include "nu/nusys.h"
 
-s32 D_80074270 = 0;
+u16 D_NEXT_HEAP_MALLOC_ID = 0;
```

Comment:
We typically name global symbols a "namespace" prefix, followed by the name in camel case, separated by an underscore. so for this I'd probably say `heap_nextMallocID` or something. Also, make sure to find and replace project-wide when renaming symbols, because there are places other than the c files that the symbols exist (symbol_addrs.txt and often undefined_syms.txt )

Reply:
I thought symbols was auto generated, fixed